docs/source/interfaces/Muon Analysis 2.rst

```diff
@@ -0,0 +1,55 @@
+.. _MuonAnalysis_2-ref:
+
+Frequency Domain Analysis
+=========================
+
+.. image::  ../images/MuonAnalysis_2.png
+   :align: right
+   :height: 400px
+
+.. contents:: Table of Contents
+  :local:
+
+Interface Overview
+------------------
+
+This interface is used to analyse the data collected on Muon instruments. The interface can be
+accessed from the main menu of MantidPlot, in *Interfaces → Muon → Muon Analysis 2*. For sample
+datasets, please visit `Muon - Downloads <http://www.isis.stfc.ac.uk/groups/muons/downloads/downloads4612.html>`_.
+
+Loading
+-------
+
+The loading section is used to load data as well as to specify what the current runs of interest are.
+
+**Load Current Run** This will load the latest run for the selected instrument. You must be connected to the
+ISIS data archive for this to work. It also currently only works on windows.
```

Comment:
This will load the latest run for the selected instrument if Mantid is connected to the archive. It currently only works on windows.